R/prettyOncoplot.R

```diff
@@ -61,60 +62,46 @@
 #' #load packages
 #' library(grid)
 #'
-#' #get some data
-#' maf_data = get_ssm_by_samples(these_samples_metadata = get_gambl_metadata())
-#' maf_metadata = get_gambl_metadata()
+#' maf_metadata <- get_gambl_metadata(seq_type_filter = "genome") %>%
+#'     dplyr::filter(pathology %in% c("FL", "DLBCL"))
 #'
-#' #define some genes of interest
-#' bl_genes = c("NFKBIZ", "ID3", "TP53", "ARID1A", "FBXO11",
-#'              "GNA13", "TCF3", "TFAP4", "HNRNPU", "FOXO1",
-#'              "CCND3", "SMARCA4", "DDX3X")
+#' maf_data <- get_ssm_by_samples(
+#'     these_samples_metadata = maf_metadata
+#' )
 #'
-#' dlbcl_genes = c("EZH2", "KMT2D", "MEF2B", "CREBBP", "MYD88")
+#define some genes of interest
+#' fl_genes = c("RRAGC", "CREBBP", "VMA21", "ATP6V1B2")
 #'
-#' genes = c(bl_genes, dlbcl_genes)
+#' dlbcl_genes = c("EZH2", "KMT2D", "MEF2B", "CD79B", "MYD88", "TP53")
 #'
-#' #define gene groups
-#' gene_groups = c(rep("BL", length(bl_genes)), rep("DLBCL", length(dlbcl_genes)))
-#' names(gene_groups) = genes
-#'
-#' #filter metadata
-#' maf_metadata = dplyr::filter(maf_metadata,!lymphgen %in% c("COMPOSITE"))
-#'
-#' #convert metadata column into factor
-#' maf_metadata$pathology = as.factor(maf_metadata$pathology)
+#' genes = c(fl_genes, dlbcl_genes)
 #'
-#' #define order of factors for selected metadata column
-#' maf_metadata$pathology = factor(maf_metadata$pathology,
-#'                                 levels = c("DLBCL", "BL",
-#'                                            "B-ALL", "CLL",
-#'                                            "COMFL", "DLBCL-BL-like",
-#'                                            "FL", "HGBL",
-#'                                            "MCL", "PBL",
-#'                                            "SCBC", "UNSPECIFIED"))
-#'
-#' maf_metadata = with(maf_metadata, maf_metadata[order(pathology),])
+#define gene groups
+#' gene_groups = c(rep("FL", length(bl_genes)), rep("DLBCL", length(dlbcl_genes)))
```

Comment:
I think there's a small typo in this line of the example. I assume you want to refer to the `fl_genes` object and not the `bl_genes`;

```
> gene_groups = c(rep("FL", length(bl_genes)), rep("DLBCL", length(dlbcl_genes)))
Error: object 'bl_genes' not found
```

Reply:
![prettyonco](https://github.com/morinlab/GAMBLR.viz/assets/32147626/3247bd7c-4809-4aba-a38a-c384eda40449)